frontend/src/modules/LoginRedux.js

```diff
@@ -55,9 +56,28 @@ function loginAction(data) {
   };
 }
 
+// getting user's information (id, first name, last name, birth date, email, username and pri or military number)
+function getUserInformation(token) {
+  return async function() {
+    let accountInfo = await fetch("/api/auth/me/", {
+      method: "GET",
+      headers: {
+        Authorization: "JWT " + localStorage.auth_token,
+        Accept: "application/json",
+        "Content-Type": "application/json",
+        cache: "default"
+      }
+    });
+    let accountInfoResponseJson = await accountInfo.json();
+    return accountInfoResponseJson;
+  };
+}
+
 // JWT tokens are not stored in our DB
 function logoutAction() {
   localStorage.removeItem("auth_token");
+  localStorage.removeItem("first_name");
```

Comment:
I don't think you need these anymore.

Reply:
Good catch!
Removed in commit #360 https://github.com/code-for-canada/project-thundercat/pull/360/commits/791e41de26cf748b6bc7cbe5eddcd7915df5b3e6